src/makeCalculator.js

```diff
@@ -5,6 +5,35 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add(num) {
+      result += num;
+    },
+    subtract(num) {
+      result -= num;
+    },
+    multiply(num) {
+      result *= num;
+    },
+    divide(num) {
+      result /= num;
+    },
```

Comment:
You can use method shorthand in object literals to define functions. Replace the function keyword with the shorthand notation for all methods (add, subtract, multiply, divide, reset, operate).